Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -1278,6 +1278,13 @@ DECLARE_SOA_COLUMN(PosZ, posZ, float);                       //! Z vertex positi
 DECLARE_SOA_COLUMN(T, t, float);                             //! Collision time relative to given bc in ns
 DECLARE_SOA_COLUMN(Weight, weight, float);                   //! MC weight
 DECLARE_SOA_COLUMN(ImpactParameter, impactParameter, float); //! Impact parameter for A-A
+DECLARE_SOA_DYNAMIC_COLUMN(GetGenId, getGenId,               //! The global generator ID which might have been assigned by the user
```

Comment:
Could you remove the comment in line 1273 and add a comment behind the //! in line 1274 that the information in GeneratorsID is encoded with o2::mcgenid::getCocktailId

Reply:
I added a comment to use the disentangled dynamic columns to access generator IDs. Do you think the comment makes sense?

Reply:
I think the comment is good. Now at second read, I wonder though why you call it `GetGenId` and not `GetGeneratorId`. I think the longer version would read better but I don't have a strong opinion.

Shouldn't we also remove the "// TODO enum to be added" because this does not make sense anymore now (at least not for the global column but only for the sub-ids...

Reply:
Thanks!